divide.py

```diff
@@ -0,0 +1,5 @@
+from .divide import divide
```

Comment:
I think the "from.divide import divide" line should be in the "pythoncalculator/init.py" file